### DISCUSSION TITLE
Unable to Access Postal Mail Server Web Interface Using Nginx

### DISCUSSION THREAD

**Discussion by sujitkhandagale** [postalserver/postal#3112] [Help with using Postal]
I'm currently experiencing an issue with accessing the Postal Mail Server web interface through my Nginx server.

I've installed the Postal Mail Server on my Ubuntu 20.04 server.
The Postal service seems to be running fine; I can see that all the services (smtp, web, worker) are running when I check the status using postal status.
Nginx is installed and running as well. I've configured Nginx according to the Postal documentation to act as a reverse proxy to the Postal web interface.

**Comment by willpower232** [comment]
What problem are you having? What do you see in your browser when you try and access the domain?

**Reply by sujitkhandagale** [reply]
i fixed my problem to access web interface.
but need more info to get smtp details and access to sending emails. i am trying to connect 465 but i am not able to do. its showing my system refused 465 connection request. need help to fix this

**Reply by willpower232** [reply]
Postal uses port 25 by default and no other ports. You can configure a different port if you like however Postal only supports explicit SSL with STARTTLS so even if you change Postals port, you may find that your clients still don't work.

If you have configured Postal to only use port 465 (or you have used something like iptables to forward packets around) then you can use `postal logs smtp` to see what your client is attempting to do. If you see no logs then it is possible that you have a firewall in place or have a typo in your Postal or client configuration.

**Comment by Dieg12** [comment]
Hello, I noticed that you have solved your problem and since I'm facing a similar issue I was wondering how did you manage to fix it. Thank you in advance.
